blocks/init/src/Blocks/components/social-links/components/social-links-editor.js

```diff
@@ -34,7 +32,7 @@ export const SocialLinksEditor = (attributes) => {
 
 		return (
 			<li className={`${componentClass}__item`}>
-				<a className={`${componentClass}__link`} href={href} title={title} dangerouslySetInnerHTML={{ __html: options.icons[icon] }} target="_blank" rel="nofollow noreferrer noopener"></a>
+				<a className={`${componentClass}__link`} href={href} title={title} dangerouslySetInnerHTML={{ __html: manifest.icons[icon] }} target="_blank" rel="nofollow noreferrer noopener"></a>
```

Comment:
can be extracted from manifest also
```
const {
icons,
} = manifest;
```
```suggestion
				<a className={`${componentClass}__link`} href={href} title={title} dangerouslySetInnerHTML={{ __html: icons[icon] }} target="_blank" rel="nofollow noreferrer noopener"></a>
```

Reply:
tnx